tests/gtests/graph/unit/backend/elyzor/test_utils.hpp

```diff
@@ -78,4 +78,104 @@ inline void construct_mul_quantize_subgraph(graph::graph_t *agraph,
     agraph->add_op(&quantize);
 }
 
+std::vector<dnnl_graph_compiler_tensor> test_tensor_to_gc_tensor(
+        std::vector<test_tensor> &in) {
+    std::vector<dnnl_graph_compiler_tensor> res;
+    res.reserve(in.size());
+
+    for (auto &test_tensr : in) {
+        auto tensr = test_tensr.get();
+        auto lt = tensr.get_logical_tensor();
+        res.emplace_back(dnnl_graph_compiler_tensor {.id = lt.id,
+                .ndims = static_cast<uint8_t>(lt.ndims),
+                .dims = lt.dims,
+                .data = tensr.get_data_handle()});
+    }
+
+    return res;
+}
+
+static void compile_execution_pipeline(impl::graph_t &agraph,
```

Comment:
this is a helper function that executes a oneDNN graph, the function was copied from [tests/gtests/graph/unit/backend/graph_compiler/test_compile_execute.cpp](https://github.com/kurapov-peter/oneDNN/blob/a9e6d220461f8ad4b2898313cf246a8ab61e7c7c/tests/gtests/graph/unit/backend/graph_compiler/test_compile_execute.cpp#L55)